pkg/client/build_test.go

```diff
@@ -1045,7 +1045,7 @@ api = "0.2"
 					h.AssertNil(t, err)
 					h.AssertEq(t, fakeLifecycle.Opts.Builder.Name(), defaultBuilderImage.Name())
 
-					bldr, err := builder.FromImage(defaultBuilderImage)
+					bldr, err := builder.FromImage(defaultBuilderImage, "", "")
```

Comment:
It would be awesome to see a test that passes in `some-value` for the run image, and then verifies that `some-value` is present in the image metadata.

Reply:
Hey! I did the changes as per the above comments, I am not sure how to write tests for this, if you could  guide me further, it would be great.